src/host/telemetry.cpp

```diff
@@ -177,40 +173,6 @@ void Telemetry::LogApiCall(const ApiCall api)
     _rguiTimesApiUsed[api]++;
 }
 
-// Log usage of the Find Dialog.
-void Telemetry::LogFindDialogNextClicked(const unsigned int uiStringLength, const bool fDirectionDown, const bool fMatchCase)
-{
-    // Don't send telemetry for every time it's used, as this will help reduce the load on our servers.
-    // Instead just create a running average of the string length, the direction down radio
-    // button, and match case checkbox.
-    _fpFindStringLengthAverage = ((_fpFindStringLengthAverage * _uiFindNextClickedTotal + uiStringLength) / (_uiFindNextClickedTotal + 1));
-    _fpDirectionDownAverage = ((_fpDirectionDownAverage * _uiFindNextClickedTotal + (fDirectionDown ? 1 : 0)) / (_uiFindNextClickedTotal + 1));
-    _fpMatchCaseAverage = ((_fpMatchCaseAverage * _uiFindNextClickedTotal + (fMatchCase ? 1 : 0)) / (_uiFindNextClickedTotal + 1));
-    _uiFindNextClickedTotal++;
-}
-
-// Find dialog was closed, now send out the telemetry.
-void Telemetry::FindDialogClosed()
-{
-    // clang-format off
-#pragma prefast(suppress: __WARNING_NONCONST_LOCAL, "Activity can't be const, since it's set to a random value on startup.")
-    // clang-format on
-    TraceLoggingWriteTagged(_activity,
-                            "FindDialogUsed",
-                            TraceLoggingValue(_fpFindStringLengthAverage, "StringLengthAverage"),
-                            TraceLoggingValue(_fpDirectionDownAverage, "DirectionDownAverage"),
-                            TraceLoggingValue(_fpMatchCaseAverage, "MatchCaseAverage"),
-                            TraceLoggingValue(_uiFindNextClickedTotal, "FindNextButtonClickedTotal"),
-                            TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES),
-                            TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage));
-
-    // Get ready for the next time the dialog is used.
-    _fpFindStringLengthAverage = 0;
-    _fpDirectionDownAverage = 0;
-    _fpMatchCaseAverage = 0;
-    _uiFindNextClickedTotal = 0;
-}

```

Comment:
It seems no one ever noticed that we passed `reverse == true` (= `upwards_search == true`) as the `fDirectionDown` parameter. In other words, the telemetry for `DirectionDownAverage` was broken. Since I feel like no one is using this telemetry anyways, I chose to remove it instead of fixing it.